src/libraries/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -1382,6 +1382,31 @@ public async Task WriteLargeToStream(bool formatted, bool skipValidation)
             Assert.Equal(expectedString, actualString);
         }
 
+        [Theory]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
+        public async Task GrowBeyondBufferSize(bool formatted, bool skipValidation)
+        {
+            const int InitialGrowthSize = 256;
+            var writer = new FixedSizedBufferWriter(InitialGrowthSize);
+            var options = new JsonWriterOptions { Indented = formatted, SkipValidation = skipValidation };
+
+            byte[] utf8String = Encoding.UTF8.GetBytes("this is a string long enough to overflow the buffer and cause an exception to be thrown.");
+
+            await using var jsonUtf8 = new Utf8JsonWriter(writer, options);
```

Comment:
Why is await needed?
```suggestion
            using var jsonUtf8 = new Utf8JsonWriter(writer, options);
```